.env

```diff
@@ -11,7 +11,7 @@ OTEL_EXPORTER_OTLP_METRICS_ENDPOINT=${OTEL_EXPORTER_OTLP_ENDPOINT}
 USERS=10
 
 # Frontend
-FRONTEND_PORT=8080
+FRONTEND_PORT=8081
```

Comment:
We need to set this to `8080`, since that will be what we use for the main Frontend UI. 8081 will be used for the FeatureFlag service UI, and 8082 for the Product catalog Admin UI.

This is per the SIG meeting on Aug-8-2022

Reply:
Hey @puckpuck I just updated it to use 8080, thanks for the review 😄